docs/sphinx/about/whats-new.txt

```diff
@@ -1,13 +1,14 @@
 Version history
 ===============
 
-5.2.0-m4 (2016 June 24)
+5.2.0-m5 (2016 July 7)
 -----------------------
 
 Top-level Bio-Formats API changes:
 
 * Java 1.7 is now the minimum supported version
 * the native-lib-loader dependency has been bumped to version 2.1.4
+* the xalan dependency has been bumped to version 2.7.2
```

Comment:
Fiji already ships `native-lib-loader` version `2.1.4`. And it does not ship `serializer` (see [scijava/pom-scijava@0814257](08142570262d44036882e4f730030b861ee63a9c) for details).

One question: does BF 5.2.0 still support Java 6? If not, then please do not upload it to the Fiji update site, but rather to the Java-8 update site. I have added `Linkert` and `Sbesson` wiki users to the list of accounts which may upload to the Java-8 site.

The only other action item is to bump the `bio-formats.version` constant in pom-scijava. You can file a PR—otherwise I will take care of it later.